### ISSUE TITLE
1.0.0 release?

### ISSUE THREAD

**Issue by doughsay** [teamon/tesla#204]
1.0.0-beta.1 has been live for almost two months, has it proven to be working enough to release 1.0.0?  Hex won't let me use a beta release as a dependency for other hex packages unless I also make them beta releases.  Just itching to upgrade :)

**Comment by teamon** [comment]
> has it proven to be working enough to release 1.0.0

This is the core question here. There was not a single bug/question reported for 1.0.0-beta1. Either the code & docs are perfect or nobody really tried to use it. I am using it myself, in production, but that's not enough.

I understand where are you standing very clearly, I would've wanted the same at your position. Unfortunately I have very limited time recently. If I release 1.0.0 with major bug it will need to be fixed fast, that's why I need to minimize the chances by publishing a beta first.

Long story short, you can make it happen quicker if you try the beta and confirm everything is fine. Or even better jump into [migration guide](https://github.com/teamon/tesla/wiki/0.x-to-1.0-Migration-Guide), point to troublesome places and report (even tiny) bugs 💪 

Thank you for starting the discussion, I will leave this issue open.

**Comment by teamon** [comment]
@ontofractal Have you used 1.0.0-beta.1 or 0.x?

**Comment by doughsay** [comment]
I will give it a shot when I can and report back!  My package isn't yet used in production, but will be within the next month.  If I can use a beta release of it using tesla 1.0 beta, I will let you know.

**Comment by ontofractal** [comment]
@teamon I've used 1.0.0-beta.1

Nothing fancy, several API clients with custom middleware so my usage of Tesla features is not extensive. Thanks for building the module, btw!

**Comment by teamon** [comment]
I've just released [v1.0.0](https://github.com/teamon/tesla/releases/tag/v1.0.0), enjoy! :)

**Comment by ontofractal** [comment]
Great news, thanks, @teamon!
